com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -46,6 +48,8 @@ public enum RpcQueueProcessingTypes
         private bool m_ProcessUpdateStagesExternally;
         private bool m_IsNotUsingBatching;
 
+        internal NetworkManager m_NetworkManager;
```

Comment:
Since this is initialized in the xtor, and never written to outside it would we not want it to be `readonly`?

Reply:
It is needed for the RpcQueueProcessor.  :)

Reply:
Does the RPCQueueProcessor need to *write* to it however?